oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -262,6 +262,15 @@ public OSProcess getProcess(int pid) {
         proc.setBytesRead(ParseUtil.parseLongOrDefault(MapUtil.getOrDefault(io, "read_bytes", ""), 0L));
         proc.setBytesWritten(ParseUtil.parseLongOrDefault(MapUtil.getOrDefault(io, "write_bytes", ""), 0L));
         
+        //gets the open files count
+        try {
+            String openFilesString = ExecutingCommand.getFirstAnswer(String.format("ls -f /proc/%d/exe | wc -l", pid));

```

Comment:
Shouldn't this be `/proc/%d/fd`?  See [proc man page](http://man7.org/linux/man-pages/man5/proc.5.html).